src/databaseScripts/CreateTables.sql

```diff
@@ -0,0 +1,75 @@
+
+USE  AgileSprints_SamB;
+DROP TABLE Employees;
```

Comment:
Is this because the guys are developing the stories already?

Reply:
Yes, these tables were created as temporary ones for different tickets.

Reply:
Okay, so we shouldn't have drops in this - so let's combine the scripts and remove the drops.